site/content/en/docs/concepts/preemption.md

```diff
@@ -0,0 +1,178 @@
+  other ClusterQueues. ClusterQueues in a cohort borrow resources in a first-come first-served fashion.
+  This algorithm is the most lightweight of the two.
+- **[Fair sharing](#fair-sharing)**: ClusterQueues with pending Workloads can preempt other Workloads in their cohort
+  until the preempting ClusterQueue obtains an equal or weighted share of the borrowable resources.
+  The borrowable resources are the unused nominal quota of all the ClusterQueues in the cohort.
+
+## Classic Preemption
+
+An incoming Workload, which does not fit within the unused quota, is eligible
+to issue preemptions when one of the following is true:
+- the requests of the Workload are below the flavor's nominal quota, or
+- `borrowWithinCohort` is enabled.
+
+### Candidates
+
+The list of preemption candidates is compiled from Workloads within the Cluster
+Queue satisfying the `withinClusterQueue` policy, and Workloads within the
+cohort which satisfy the `reclaimWithinCohort` policy.
+
+The list of candidates is sorted based on the following preference checks for
+tie-breaking:
+- Workloads from borrowing queues in the cohort,
+- Workloads with the lowest priority,
+- Workloads which got admitted the most recently.
+
+### Targets
+
+The algorithm qualifies the candidates as preemption targets using the heuristics
+below:
+
+1. If all candidates belong to the target queue, then Kueue greedily
+qualifies candidates until the preemptor Workload can fit, allowing the usage of
+the ClusterQueue to be above the nominal quota, up to the `borrowingLimit`.
+This is referred as "borrowing" in the points below.
+
+2. If `borrowWithinCohort` is enabled, then Kueue greedily qualifies
+candidates (respecting the `borrowWithinCohort.maxPriorityThreshold` threshold),
+until the preemptor Workload can fit, allowing for borrowing.
+
+3. If the current usage of the target queue is below nominal quota, then
+Kueue greedily qualifies the candidates, until the preemptor Workload can fit,
+disallowing for borrowing.
+
+4. If the Workload didn't fit by using the previous heuristics, Kueue greedily
+qualifies only the candidates which belong to the preempting Cluster Queue,
+until the preemptor Workload can fit, allowing for borrowing.
+
+The last step of the algorithm is to minimize the set of targets. For this
+purpose, Kueue greedily traverses the list of initial targets in reverse and
+removes a Workload from the list of targets if the preemptor Workload still can be
+admitted when accounting back the quota usage of the target Workload.
+
+## Fair Sharing
+
+{{% alert title="Note" color="primary" %}} 
+Available in Kueue v0.7.0 and newer
+{{% /alert %}} 
+
+To enable fair sharing, [use a Kueue Configuration](/docs/installation#install-a-custom-configured-release-version) similar to the following:
+
+```yaml
+apiVersion: config.kueue.x-k8s.io/v1beta1
+kind: Configuration
+fairSharing:
+  enable: true
+  preemptionStrategies: [LessThanOrEqualToFinalShare, LessThanInitialShare]
+```
+
```

Comment:
The attributes in this Kueue configuration are described in the following sections.

Reply:
Added